### PR TITLE
docs(self-managed): add multi-namespace deployment guide

### PR DESCRIPTION
**Pull request by aabouzaid**
## Description

Highlight the changes needed for multi-namespace deployment which will be part of 8.4 release.

## When should this change go live?

ASAP

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] They are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`).
- [x] My changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

**Comment by christinaausley** [comment]
Hi @aabouzaid -- is this still in draft because you are planning to make additional changes? Let me know when you are ready for final review 👍

**Comment by aabouzaid** [comment]
@christinaausley it's ready for review, thanks a lot :raised_hands:

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/multi-namespace-deployment.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+id: multi-namespace-deployment
+title: "Multi-namespace deployment"
+description: "Deploy Camunda 8 Self-Managed across namespaces"
```
Comment (christinaausley) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/multi-namespace-deployment.md:4]: I would add a purpose here, so "Deploy Camunda 8 Self-Managed across namespaces so you can..."

Essentially, I'm just wondering what the benefit is so users may feel more inclined to click into the documentation.

**Review by christinaausley — COMMENTED** [review]
Just need to add this into the sidebar files -- thank you!

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/multi-namespace-deployment.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+You can deploy Camunda 8 Self-Managed across namespaces, where we have a single management deployment (Identity and Web Modeler), and multiple automation deployments (Zeebe, Operate, Tasklist, Optimize).
+
+In this setup each namespace has its own Helm deployment independently, but each one uses a different values file based on if the deployment mode is `management` or `automation`.
+
+The following are three values files, one for the `managment` deployment and two for the `automation` deployment:
+
+## Management deployment
+
+This deployment works with Identity and Web Modeler only and allows another two deployments to authenticate against Keycloak:
```
Comment (christinaausley) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/multi-namespace-deployment.md:15]: "and allows another two deployments to authenticate against Keycloak" -- I adjusted the wording here. Is this what you meant?

**Review by HamzaMasood1 — APPROVED** [review]
👍

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/multi-namespace-deployment.md (hunk 1 above)
Comment (aabouzaid) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/multi-namespace-deployment.md]: I've added more details in the description :+1:

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/multi-namespace-deployment.md (hunk 2 above)
Comment (aabouzaid) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/multi-namespace-deployment.md]: @christinaausley  The meaning is a bit different, so I've reworded it.
Could you please give it another look?

**Comment by christinaausley** [comment]
Let me know if you need assistance getting this into the sidebar file and we can merge 👍

**Comment by aabouzaid** [comment]
@christinaausley added :heavy_check_mark:

**Review by christinaausley — APPROVED** [review]

**Pull request merged**